### DISCUSSION TITLE
NAS option not available to select music

### DISCUSSION THREAD

**Discussion by cYCL157** [rern/rAudio#1644] [Fixed Issues]
I just updated to rAudio 20240330 from 20240316 and after updating and rebooting, am not able to get my music to select something else to listen to.  The playlist that I had playing when I updated is still playing and the system tab shows my mounts are still available and successfully mounted.  See attached images

https://i.imgur.com/Zq8tOc5.png

https://i.imgur.com/Wl1ztTm.png

**Comment by markblinkhorn** [comment]
Same here. I have a legacy 32 bit Pi2 and also a 64bit Pi3 - both of which do the same. NAS icon in the library is greyed out and will not open. I can ls the NAS folders from the CLI. No clues in journalctl.

I restored the backup that I took immediately prior to the update. This has not resolved the issue. The NAS icon is still inpoerable.

**Comment by markblinkhorn** [comment]
Here's a workaround:

I copied /srv/http/mpdlibrary.php from a working installation to a broken one. The NAS icon lights up again and I can select new choons to play.

**Comment by beonun** [comment]
Same here.
I can't mount NAS
And in library I don't know why, certain folder subfolders won't open.
Therefore, even though I can add a music file included in the folder, I cannot click to view the folder.

**Comment by rern** [comment]
Today's update should fix it.

**Comment by markblinkhorn** [comment]
I can confirm today's update works for me.

Mark

@rern  - UPDATE: But traversing the NAS directory structure (Windows share attached to a Synology NAS iSCSI volume) is very slow once more.

**Reply by markblinkhorn** [reply]
rAudio 20240406 fixes slow traversal of NAS folders - thanks @rern

**Comment by Fixu69** [comment]
No. Even after update 20240331 and reboot, nothing happens when clicking on NAS...

Todays update 20240406 has solved the Problem. thanks @rern!

**Comment by cYCL157** [comment]
Mine is working again, thanks @rern!

**Comment by R101G** [comment]
@rern 
I see the same problem after using Restore on a new image. Backup/Restore has not restored the mount folder in /mnt/MPD/NAS/
